docs/terms/evaluation_parameter.md

```diff
@@ -1,15 +1,132 @@
 ---
 id: evaluation_parameter
 title: Evaluation Parameter
-hoverText: A parameter used during Validation of an Expectation to reference simple expressions or previousl generated metrics.
+hoverText: A connector to store and retrieve information about parameters used during Validation of an Expectation which reference simple expressions or previously generated metrics.
 ---
+
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
+
+An Evaluation Parameter is a connector to store and retrieve information about parameters used during <TechnicalTag relative="../" tag="validation" text="Validation" /> of an <TechnicalTag relative="../" tag="expectation" text="Expectation" /> which reference simple expressions or previously generated metrics.
```

Comment:
again, since `connector` happens to be one of technical terms, i think this might be confusing. Off the top of my head, I would recommend `a dynamic value`, but i'm sure there are other options as well.